.expeditor/config.yml

```diff
@@ -45,3 +45,5 @@ pipelines:
       definition: .expeditor/website.pipeline.yml
       trigger: pull_request
   - habitat/build
+  - habitat_release:
```

Comment:
This is more for my edification than anything, but what's the distinction between `habitat/build` (with a `/`) and `habitat_release` (with a `_`)?

Reply:
Good eye - `habitat/build` defines a expeditor "habitat" type pipeline called "build". In this instance `habitat_release` is the name of a "generic" pipeline. More info here https://expeditor.chef.io/docs/integrations/buildkite/#supported-pipeline-trigger-types